docs/How  to use the epos plugin.md

```diff
@@ -0,0 +1,136 @@
+# How to use the Epos plugin of the FAIR evaluator 
+To use the fair evalater you need access to a terminal 
+
+## 1.Follow the steps indicated in the  documentation
+https://github.com/EOSC-synergy/FAIR_eva/blob/main/docs/index.md which are executting the following commands
```

Comment:
Use markdown link for this; URLs are ugly ;)

Reply:
Upgraded